cluster/node-pools/master-default/userdata.clc.yaml

```diff
@@ -408,6 +408,10 @@ storage:
             - --requestheader-username-headers=X-Remote-User
             - --proxy-client-cert-file=/etc/kubernetes/ssl/proxy-client.pem
             - --proxy-client-key-file=/etc/kubernetes/ssl/proxy-client-key.pem
+            # kubelet authentication
```

Comment:
sorry for dropping in, I can't figure out the way you guys/girls rotate master/worker nodes when a change to user-data happens. I saw the node-drain unit, but can't understand how the rotation is triggered. Is there any public documentation?

Reply:
This is all handled by our Cluster Lifecycle Manager: https://github.com/zalando-incubator/cluster-lifecycle-manager

Reply:
Thank you @hjacobs, i'll take a look at it